dependencies/src/test/java/com/google/cloud/tools/opensource/classpath/LinkageCheckReportTest.java

```diff
@@ -111,4 +118,52 @@ public void testCreation() {
             .getReference()
             .getTargetClassName());
   }
+
+  @Test
+  public void testConversion() throws IOException, URISyntaxException {
+    Path path1 = absolutePathOfResource("testdata/grpc-google-cloud-firestore-v1beta1-0.28.0.jar");
+    Path path2 = absolutePathOfResource("testdata/google-cloud-firestore-0.65.0-beta.jar");
+    SymbolProblem symbolProblem1 =
+        new SymbolProblem(
+            new ClassSymbol("java.lang.Integer"),
+            ErrorType.CLASS_NOT_FOUND,
+            new ClassFile(Paths.get("foo", "bar.jar"), "java.lang.Object"));
+    SymbolProblem symbolProblem2 =
+        new SymbolProblem(
+            new MethodSymbol("java.lang.Integer", "toString", "Ljava.lang.String;", false),
+            ErrorType.INACCESSIBLE_MEMBER,
+            new ClassFile(path2, "java.lang.Object"));
+    ImmutableSetMultimap<ClassFile, SymbolProblem> symbolProblems =
+        ImmutableSetMultimap.of(
+            new ClassFile(path1, "com.google.firestore.v1beta1.FirestoreGrpc"), symbolProblem1,
+            new ClassFile(path2, "com.google.ClassB"), symbolProblem2);
+
+    SymbolReferenceMaps.Builder referenceMapBuilder = new SymbolReferenceMaps.Builder();
+    LinkageCheckReport linkageCheckReport =
+        LinkageCheckReport.fromSymbolProblems(
+            symbolProblems,
+            ImmutableList.of(path1, path2),
+            ClassReferenceGraph.create(referenceMapBuilder.build(), ImmutableSet.of(path1)));
+    Truth.assertThat(linkageCheckReport.getJarLinkageReports()).hasSize(2);
```

Comment:
Truth doesn't seem to add anything compared to assertEquals in this method, and bloats the code a bit.